css/indigo3.css

```diff
@@ -5750,7 +5750,16 @@ CRM form embed
   cursor: pointer; }
 
 .paragraphs-item-msd-form-embed #ref_guide {
-  display: none; }
+  cursor: pointer;
+  color: #0055F0;
+  position: relative;
+  left: 0.2em;
```

Comment:
Do we use `em` at ucl? Feel like I've never noticed it before

Reply:
Hi we usually use rem - it's more reliable on page zoom.  The value will be the same for both though and for a little value like this it probably won't matter.